src/main/java/dev/ebullient/convert/tools/dnd5e/Json2QuteClass.java

```diff
@@ -905,9 +906,8 @@ void addSpellSlot(String value) {
         }
 
         String toHtmlLink(String x, String level) {
-            return String.format("<a href='#%s'>%s</a>",
-                    toAnchorTag(x + " (Level " + level + ")"),
-                    x);
+            return String.format("<a href='#%s'class=\"internal-link\">%s</a>",
```

Comment:
HTML doesn't care about single or double quotes, and this is a little easier to read. Also, missed a space between href & class.

```suggestion
            return String.format("<a href='#%s' class='internal-link'>%s</a>",
```